app/src/main/java/com/duckduckgo/app/cta/ui/CtaViewModel.kt

```diff
@@ -352,6 +357,8 @@ class CtaViewModel @Inject constructor(
 
     private fun isSerpUrl(url: String): Boolean = url.contains(DaxDialogCta.SERP)
 
+    private fun isDuckDuckGoEmailUrl(url: String): Boolean = url.contains(DUCK_DUCK_GO_EMAIL_URL_PART)
```

Comment:
Rather than doing your own implementation of this method you can reuse `isDuckDuckGoEmailUrl` from `DuckDuckGoUrlDetector` which should be a bit more safe.